docs/development.md

```diff
@@ -11,98 +11,126 @@ See [the pull request template](../.github/pull_request_template.md) for steps t
+
+You can use the Git commands below BEFORE cloning the repository to your local computer to prevent files from using `CRLF` by default.
+```
+# don't use --global if you only want this behavior for a specific repository.
+git config --global core.autocrlf false
+# this ensures all files use LF.
+git config --global core.eol lf
+```
+---
 
-## Environment Variables
+### Environment Variables
 
 Create a .env file in the `/backend` directory.
 Add and define the following environment variables using the instructions below.
 
 ```
 ENV = 'LOCAL'
-SAM_API_KEY =
-SECRET_KEY =
-DJANGO_SECRET_LOGIN_KEY =
-LOGIN_CLIENT_ID =
-DISABLE_AUTH = 
+DISABLE_AUTH = False
+SECRET_KEY = YourSecretKey
+CLAMAV_PORT = 
+SAM_API_KEY = 
+DJANGO_SECRET_LOGIN_KEY = 
+LOGIN_CLIENT_ID = 
 ```
 
-For local testing, you may need to specify a few other variables:
-
-* A port other than `9000` for clamav-rest: add `CLAMAV_PORT = {port number}` to your `.env` file.
-* Cypress variables for running local end-to-end tests. See the [testing docs](https://github.com/GSA-TTS/FAC/blob/main/docs/testing.md#end-to-end-testing) for more.
-
-If you need to add these to your local environment (should end up in `~/.bash_profile`, `~/.bashrc`, `~/.zshrc`, or whatever flavor of shell you're using.)
-
 #### ENV
 The `ENV` environment variable specifies the set of configuration settings to use while running. For local development, it should be `LOCAL`, which will enable settings that should work on your local machine with Docker.
 
 On our Dev/Staging/Production environments, it will be set to `DEVELOPMENT`/`STAGING`/`PRODUCTION` respectively. Setting it to one of these will turn on configuration it's expecting while deployed to Cloud.gov.
 
 In GitHub Actions and our CI/CD pipeline, it is set to `TESTING`.  It will enable settings expected to make unit tests complete properly while still trying to emulate a Cloud.gov situation.
 
-While you can change this, you generally shouldn't need to.
+While you **could** change this, you generally shouldn't need to.
+
+#### DISABLE_AUTH
+The `DISABLE_AUTH` variable tells Django to disable the Login.gov authorization. This should almost always be `False` unless you need to temporarily disable it for your local development. 
+
+In the Dev/Staging/Production environments, it will be set to `False` and require all users to go to Login.gov to log in.
+
+In GitHub Actions (the CI/CD pipeline), it will be set to `True` to complete unit testing and frontend testing properly.
+
+#### SECRET_KEY
+Generate your own random secret key for local development. Django uses this to provide cryptographic signing.
+
+#### CLAMAV_PORT
+A port other than `9000` for the clamav-rest image.
```

Comment:
I know this isn't your verbiage, but I think it could be made more clear that this var is only needed if you don't want clamav running on 9000 (because that port is already in use)

Reply:
In my head I thought it was preferred that this is set to a different port. But so long as that is optional and not the recommended way to do things, I can totally put this language back where it originally was (in the section with the Cypress variables).